README.md

```diff
@@ -37,7 +37,7 @@ many users will want iris and/or xarray as well:
 
 
 ```
-conda install -c conda-forge -c ioam holoviews>=1.6.2 geoviews
```

Comment:
Shouldn't it still enforce *some* minimal compatiable version of hv?  Or are you counting on conda automatically suggesting updates anyway?